src/psyclone/tests/psyir/nodes/structure_reference_test.py

```diff
@@ -241,8 +241,8 @@ def test_struc_ref_datatype():
     grid_type = symbols.StructureType.create([
         ("nx", symbols.INTEGER_TYPE, symbols.Symbol.Visibility.PUBLIC, None),
         ("data", atype, symbols.Symbol.Visibility.PRIVATE, None),
-        # A single member of structure type.
-        ("roger", rtype, symbols.Symbol.Visibility.PUBLIC, None),
+        # A single member of structure type with a mixed-case name.
+        ("roGEr", rtype, symbols.Symbol.Visibility.PUBLIC, None),
         # An array of structure type.
         ("titty", artype, symbols.Symbol.Visibility.PUBLIC, None)])
```

Comment:
I'll leave this up to you - but maybe some renaming of arguments/variables in this file.

Reply:
Yes, fair point. It was a Swallows and Amazons naming scheme but that is probably too obscure :-)

Reply:
Ah ok, I did not know the reference so was a little confused!